package.json

```diff
@@ -105,10 +111,7 @@
     "ace-builds": "^1.4.12",
     "bufferutil": "4.0.1",
     "create-react-class": "15.6.2",
-    "electron": ">=7.2.4",
     "electron-json-storage": "4.0.2",
-    "eslint": "^7.5.0",
-    "eslint-plugin-jsdoc": "^30.2.1",
```

Comment:
These are basically devDependencies and don't need to be dependencies for the actual app, this way we save space for the size of the packaged app.